clang/tools/clang-format/clang-format-diff.py

```diff
@@ -102,9 +102,9 @@ def main():
     filename = None
     lines_by_file = {}
     for line in sys.stdin:
-        match = re.search(r"^\+\+\+\ (.*?/){%s}(\S*)" % args.p, line)
+        match = re.search(r"^\+\+\+\s+(?:.*?/){%s}(.+)$" % args.p, line)
```

Comment:
```suggestion
        match = re.search(r"^\+\+\+\ (.*?/){%s}(.+)" % args.p, line.rstrip())
```
and restore line 107 below to make the minimum change.

Reply:
Made the change suggested, tested, working good. Messed up the PR with force push sorry about that.